static/javascript/realtime.js

```diff
@@ -141,5 +145,15 @@ function updateHead(chart) {
     let header = document.getElementById(head_key);
     header.innerText = latest_val;
 
-
+	if(latest_val > db_format[data_key]["safe_max"] || latest_val < db_format[data_key]["safe_min"]) {
```

Comment:
FYI, as an alternative to if condition add else remove, there is also a classList.toggle which takes an optional boolean as a second parameter (e.g. `element.classList.toggle("class", condition)`)